### ISSUE TITLE
Consider adding `django-drifter`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-django-template#2717]
https://github.com/kennethlove/django-drifter

**Comment by DmytroLitvinov** [comment]
@sobolevn , I suppose you also subscribed to Django-Newsletter? :)

**Comment by sobolevn** [comment]
Yes :)

I really need the `reset_db` command, because right now I use `psql` with a custom script :(

**Comment by sobolevn** [comment]
This clearly should be a dev dependency.

**Comment by anywindblows** [comment]
`django-drifter` has not been updated for 7 months. Is it really necessary?

**Comment by sobolevn** [comment]
yes, because `django-drifter` proved to be a really great tool in development. it has nothing to be updated.
